### PR TITLE
DOC: Improve inverted axis example

### LINKED ISSUE #28050: [Doc]: Invert Axes example sets x-lims directly
### Documentation Link

https://matplotlib.org/devdocs/gallery/subplots_axes_and_figures/invert_axes.html

### Problem

This example shows how to invert the x-axis by using `ax.set_xlim`.  I would have expected it to use the `set_inverted` method on the x-axis: that is more flexible as it doesn't prevent autoscaling on subsequent calls.

I note that this example pre-dates the addition of `set_inverted` to the library.

### Suggested improvement

Either change the existing code to use `set_inverted` or add a second figure within the example to show its use.

### PR DESCRIPTION
**Pull request by timhoffm**
Closes #28050.

### CODE REVIEW

**Review by rcomer — APPROVED** [review]
Thanks for picking this up!  I had been going back and forth on whether and how to still include the `set_xlim` version, but this solves it very neatly.

I had been thinking along similar lines to @story645 to put two figures in with some explanation in between, but all I could come up with for when to use `set_xlim` was "if this is a context where you would have fixed the limits regardless of direction".  That doesn’t really warrant a whole paragraph and the new axes titles cover this I think.

I’m approving as definitely better than it was but will also leave open for a bit in case others want to discuss further.

**Comment by rcomer** [comment]
> 2) provides a way to more clearly identify a preferred/recommended method. I feel like putting em both in the same figure denotes that they're at the same level.

When I opened the issue I was obviously thinking `set_inverted` is the "right" way to do this but actually if you want fixed limits then using `set_xlim` would be the right way.  So I'm not sure I'd say one should be preferred in general, but that it depends on your use-case.

**Comment by timhoffm** [comment]
I still believe one figure is better, because you have the two code variants right next to each other, which my ales it easier to compare.

we can however make the “two variants” situation more visible in the description.

Will update later.

**Comment by rcomer** [comment]
> But here the image is identical

They are not identical, as the fixed limit version has narrower limits.  Perhaps that would be more obvious if they were one above each other instead of side-by-side?

**Comment by timhoffm** [comment]
Let's discuss based on this version.

File: galleries/examples/subplots_axes_and_figures/invert_axes.py [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
-===========
-Invert Axes
-===========
+=============
+Inverted axis
+=============
 
-You can use decreasing axes by flipping the normal order of the axis
-limits
+There are two ways to invert the direction of an axis:
```
Comment (rcomer) [inline galleries/examples/subplots_axes_and_figures/invert_axes.py:6]:
```suggestion
This example demonstrates two ways to invert the direction of an axis:
```
My turn to be pedantic!  There are [many ways](https://stackoverflow.com/questions/2051744/how-to-invert-the-x-or-y-axis/78298718#78298718), but I think only these two are worth highlighting.

**Comment by timhoffm** [comment]
Sort of the point here is that both methods yield similar results: Both have inverted x-axis. The difference fixed vs. autoscaled axes is not immediately visible. We could use a larger range for the fixed-limits case, so that one clearly sees these are not autoscaled, but do we want to draw attention to this?

On one or two plots:
With two separate plots, the code blocks and figures are further apart and thus harder to compare. IMHO this is a disadvantage no matter whether we want to emphasize both variants result in similar plots, or whether we want to emphasize the difference in limits from auto-scaling vs. fixed limits.

**Comment by timhoffm** [comment]
Used x, y as variables and "decreasing x ⟶" to draw more attention to the x axis.

To put even more emphasis, we could color xticks/xlabel e.g. orange, but I'm hesitant that the additional commands add too much clutter.

**Review by rcomer — APPROVED** [review]

**Comment by timhoffm** [comment]
@story645 @jklymak I yield here. My motivation and scope was a reasonable quick fix for #28050, which I believe is fulfilled. Feel free to bikeshed further without me.

**Comment by rcomer** [comment]
Let's get this in...

**Pull request merged**